traffic_ops/traffic_ops_golang/deliveryservice/acme.go

```diff
@@ -347,13 +409,19 @@ func GetAcmeCertificates(cfg *config.Config, req tc.DeliveryServiceLetsEncryptSS
 	if err != nil {
 		log.Errorf("acme: getting acme client for provider %s", provider)
```

Comment:
Because this is being called as a goroutine, the returned error is discarded. Because of this, and because this print statement does not include the error message, if this error occurs operators will have no idea what caused it or what to do to rectify the issue.

At a glance this _looks_ like the only place where this kind of error swallowing occurs, but I'd appreciate if you'd look it over and make sure I didn't miss anything.

Alternatively, if you wanted, you could cut down on the log statements by making `GetAcmeCertificates` a wrapper with a panic handler that gets an error from something unexported like e.g. `getACMECertificates` that just _returns_ an error, and then the wrapper could log it.

But that's a pretty big structural change and afaict if you just append the error message here it won't be an actual problem, so you don't have to.